arm/compute/models.go

```diff
@@ -397,7 +374,6 @@ type AvailabilitySet struct {
 	Location                   *string             `json:"location,omitempty"`
 	Tags                       *map[string]*string `json:"tags,omitempty"`
 	*AvailabilitySetProperties `json:"properties,omitempty"`
-	Sku                        *Sku `json:"sku,omitempty"`
```

Comment:
we're currently reliant on this field to expose whether an Availability Set is Managed or not - has it been Deprecated?